presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -7043,9 +7043,9 @@ public void testArrayShuffle()
     @Test
     public void testNonReservedTimeWords()
     {
-        assertQuery("" +
-                "SELECT TIME, TIMESTAMP, DATE, INTERVAL\n" +
-                "FROM (SELECT 1 TIME, 2 TIMESTAMP, 3 DATE, 4 INTERVAL)");
+        assertQuery(
```

Comment:
What is this change about?

Reply:
- In H2 1.4.199, `INTERVAL` became a reserved word, and hence this original query would fail in H2.
- This test was to ensure those 4 words are not reserved words in Presto. Therefore, we just need to make sure that the original query can still be ran on Presto and it can produces the correct results.